diff --git a/build.gradle b/build.gradle
--- a/build.gradle
+++ b/build.gradle
@@ -16,6 +16,7 @@ repositories {
 	// Loom adds the essential maven repositories to download Minecraft and libraries from automatically.
 	// See https://docs.gradle.org/current/userguide/declaring_repositories.html
 	// for more information about repositories.
+	mavenLocal()
 	maven {
 		url = "https://www.cursemaven.com/"
 	}
@@ -47,7 +48,7 @@ dependencies {
 	modImplementation "curse.maven:${project.modmenu_version}"
 
 	//3d skin layers
-	modImplementation "curse.maven:${project.skinlayers_version}"
+	modImplementation group: 'dev.tr7zw', name: 'skinlayers', version: '1.4.7-mc1.19'
 
 	//multiconnect TODO readd when updated for 1.18.2
 	//modImplementation "curse.maven:${project.multiconnect_version}"
